deck/models.py

```diff
@@ -42,7 +42,7 @@ class Deck(models.Model):
     include_jokers = models.BooleanField(null=False, default=False)
     
     def open_new(self, cards_used=None, jokers_enabled=False):
-        self.include_jokers = jokers_enabled
+        self.include_jokers = False if jokers_enabled is None else jokers_enabled
```

Comment:
Is there a reason we need to all this and we can't just say 
`self.include_jokers = jokers_enabled`, and then we can change the `get_jokers_enabled` to always return a boolean value?

Reply:
See my comment on `get_jokers_enabled` for a broader explanation, but I basically moved the assumption that if a request doesn't specify jokers, that jokers should not be included to this declaration.

Reply:
Okay, I see how it's working now.